File: payment-service/src/main/java/com/hedvig/paymentservice/domain/trustlyOrder/commands/CancelNotificationReceivedCommand.java
package com.hedvig.paymentservice.domain.trustlyOrder.commands;

import java.util.UUID;

import org.axonframework.commandhandling.TargetAggregateIdentifier;

import lombok.Value;

@Value
public class CancelNotificationReceivedCommand {
    @TargetAggregateIdentifier
    UUID hedvigOrderId;

    String notificationId;
    String trustlyOrderId;
    String memberId;
}

File: payment-service/src/main/java/com/hedvig/paymentservice/domain/trustlyOrder/commands/CancelNotificationReceivedCommand.kt
package com.hedvig.paymentservice.domain.trustlyOrder.commands

import java.util.UUID
import org.axonframework.commandhandling.TargetAggregateIdentifier

data class CancelNotificationReceivedCommand(
    @TargetAggregateIdentifier
    val hedvigOrderId: UUID,

    val notificationId: String,
    val trustlyOrderId: String,
    val memberId: String
)
